### DISCUSSION TITLE
unreported error: x-if on div

### DISCUSSION THREAD

**Discussion by markfirmware** [alpinejs/alpine#977] [4. General]
https://codepen.io/markfirmware/pen/MWjGgRj

**Comment by KevinBatdorf** [comment]
The console has other errors in there that you need to address first, then the `x-if` warning will show (along with another error)

**Reply by markfirmware** [reply]
what console?

**Reply by KevinBatdorf** [reply]
The browser console. See here: https://support.airtable.com/hc/en-us/articles/232313848-How-to-open-the-developer-console
